modules/common/wikiquote.js

```diff
@@ -7,135 +7,139 @@ var request = require.safe('request');
  * Added by Jay Harris on 22/07/2015
  */
 (function() {
-  var API_URL = "http://en.wikiquote.org/w/api.php";
+    var API_URL = 'http://en.wikiquote.org/w/api.php';
 
-  /**
-   * Query based on "titles" parameter and return page id.
-   * If multiple page ids are returned, choose the first one.
-   * Query includes "redirects" option to automatically traverse redirects.
-   * All words will be capitalized as this generally yields more consistent results.
-   */
-  exports.queryTitles = function(titles, success, failure) {
-    var url = API_URL + "?format=json&action=query&redirects=&titles=" + titles;
-    request.get(url,
-      function(error, response, body) {
-        if (error || response.statusCode != 200) {
-          failure("No results");
-          return;
-        }
-        result = JSON.parse(body);
-        var pages = result.query.pages;
+    /**
+    * Query based on "titles" parameter and return page id.
+    * If multiple page ids are returned, choose the first one.
+    * Query includes "redirects" option to automatically traverse redirects.
+    * All words will be capitalized as this generally yields more consistent results.
+    */
+    exports.queryTitles = function(titles, success, failure) {
+        var url = API_URL + '?format=json&action=query&redirects=&titles=' + titles;
+        request.get(url,
+            function(error, response, body) {
+                if (error || response.statusCode !== 200) {
+                    failure('No results');
+                    return;
+                }
+                var result = JSON.parse(body),
+                    pages = result.query.pages,
+                    pageId = -1;
 
-        var pageId = -1;
-        for(var p in pages) {
-          var page = pages[p];
-          // api can return invalid recrods, these are marked as "missing"
-          if(!("missing" in page)) {
-            pageId = page.pageid;
-            break;
-          }
-        }
-        if(pageId > 0) {
-          success(pageId);
-        } else {
-          failure("No results");
-        }
-      });
-  };
+                for (var p in pages) {
+                    var page = pages[p];
+                    // api can return invalid recrods, these are marked as "missing"
+                    if(!('missing' in page)) {
+                        pageId = page.pageid;
+                        break;
+                  }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 20 space characters but found 18.](https://www.codacy.com/app/theprojectshop/Kassy/file/2651361737/issues/source?bid=3282987&fileBranchId=3324570#l36)